sirepo/http_util.py

```diff
@@ -0,0 +1,48 @@
+"""Support routines for http requests/responses
+
+:copyright: Copyright (c) 2024 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdlog
+import re
+import sirepo.job
+
+#: Http auth header name
+_AUTH_HEADER = "Authorization"
+
+#: http auth header scheme bearer
+_AUTH_HEADER_SCHEME_BEARER = "Bearer"
+
+#: Regex to test format of auth header and extract token
+_AUTH_HEADER_RE = re.compile(
+    _AUTH_HEADER_SCHEME_BEARER + r"\s(" + sirepo.job.UNIQUE_KEY_CHARS_RE + ")",
```

Comment:
The `\s` should be `\s+` (see [rfc6750](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1)). unique_key should be moved to sirepo.util. The connection to job is confusing here.

TODO: create_token should be aligned with unique_key. They use two different sizes, but that's too complicated (involves db upgrade). create_token should be fixed to check `value`.

Reply:
390567377

> create_token should be fixed to check value

What do you mean by this?

Reply:
NVM. Confusion on my part. value must be passed to util.create_token. I thought it could be None in some cases, because there's AuthUserEmail.create_token() which takes no args